api/src/main/java/org/pmiops/workbench/db/model/DbDemographicSurvey.java

```diff
@@ -35,30 +35,7 @@ public class DbDemographicSurvey {
   private List<Short> race;
   private List<Short> genderIdentityList;
   private List<Short> sexAtBirth;
-  private int year_of_birth;
-
-  public DbDemographicSurvey() {}
```

Comment:
Because you got rid of the custom constructor, does that mean that the empty constructor is added automatically and does not need to be added implicitly?

Reply:
yep